core/src/main/scala/com/fortysevendeg/mvessel/Driver.scala

```diff
@@ -71,12 +71,11 @@ class Driver extends SQLDriver {
 
 object Driver {
 
-  Try {
-    java.sql.DriverManager.registerDriver(new Driver)
-  } match {
-    case Failure(e) => e.printStackTrace()
-    case _ =>
-  }
+  def registerDriver() =
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [You should not name methods after their object](https://www.codacy.com/app/47deg/mvessel/file/848926616/issues/source?bid=2442814&fileBranchId=2811084#l74)